src/components/Brush.js

```diff
@@ -150,6 +158,11 @@ export default React.createClass({
             initialBrushEndTime: null,
             initialBrushXYPosition: null
         });
+		
+        if (this.props.onSelectComplete) {
+            this.props.onSelectComplete();
+            this.props.onTimeRangeChanged(null);
```

Comment:
hmm... I think onTimeRangeChanged should be separately protected here. If onSelectComplete is passed in as a prop, but onTimeRangeChanged is not, this will break.